pages/slice/language-guide/class-types.md

```diff
@@ -178,6 +178,22 @@ When the generated code decodes a class in sliced format and slices off derived
 slices: it keeps them in the decoded instance. If you later send this class instance to another application that knows
 the derived class type, this application will decode successfully the full type.
 
+## Compact type IDs
+
+When the generated code encodes a class it needs a way to transmit the class's type.
+It does this by encoding the class's _type ID_ - a special value that is unique to each class.
+Normally, this is the class's fully-qualified identifier, which gets encoded as a string.
+However, to reduce overhead, Slice allows you to assign a numeric type ID to a class instead. For example:
+
+```slice
+class ClassWithStringTypeId {}     // has a type ID of "ClassWithStringTypeId"
+class ClassWithNumericTypeId(9) {} // has a type ID of `9`
+```
+
+Compact type IDs have no effect on the mapping of a class, or the contract for using it.
+It only affects how the class is encoded and decoded.
```

Comment:
The serialized format is called "the encoding".

Reply:
I replaced "how the class is encoded and decoded" with "the encoding of classes".